apps/docs/content/components/Card/index.mdx

```diff
@@ -82,21 +82,21 @@ Cards typically appear in groups of three or four in desktop viewports. For smal
 
 ## Anatomy
 
-![An image showing the card anatomy with an icon as a visual asset, a heading, a description and a CTA text in a vertical order.](https://github.com/primer/brand/assets/912236/633b83d8-4ea8-49bd-9d6c-45a1989060d9)
+![An image showing the card anatomy with an icon as a visual asset, a heading, a description and a CTA text in a vertical order.](https://github.com/primer/brand/assets/6951037/61039c70-6dc8-4357-8859-8299bdad78fb)
```

Comment:
This one doesn't show any image, it seems to be a link to a video

<img width="692" alt="Screenshot 2023-10-06 at 12 00 07" src="https://github.com/primer/brand/assets/912236/2941ed53-91c8-4919-bdd0-281938f13ee5">